nc/nc.go

```diff
@@ -474,3 +476,36 @@ func (c *Client) Stop() error {
 
 	return nil
 }
+
+// Revoke revokes the NC communication token
+func (c *Client) Revoke(purgeSession bool) bool {
+	c.startMu.Lock()
+	defer c.startMu.Unlock()
+
+	if c.started {
+		log.Println(logPrefix, "attempt to revoke token for running client")
+		return false
+	}
+
+	var cfg config.Config
+	if err := c.credsFetcher.cm.Load(&cfg); err != nil {
+		log.Println(logPrefix, "error reading cfg:", err)
+		return false
+	}
+	userID := cfg.AutoConnectData.ID
+	tokenData := cfg.TokensData[userID]
+
+	log.Println(logPrefix, "revoking token")
+	resp, err := c.credsFetcher.api.NotificationCredentialsRevoke(tokenData.Token, tokenData.NCData.UserID.String(), purgeSession)
+	if err != nil {
+		log.Println(logPrefix, "error revoking token:", err)
+		return false
+	}
+	if strings.ToLower(resp.Status) == "ok" {
+		log.Println(logPrefix, "revoked token")
+		return true
+	} else {
+		log.Println(logPrefix, "not revoked token")
```

Comment:
for error wouldn't make sense to log the `Status`?

Reply:
Done.